assets/js/components/text-toolbar-button/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function TextToolbarButton( props ) {
+	const classes = classnames(
+		'wc-block-text-toolbar-button',
+		props.className
+	);
+	return (
+		<Button className={ classes } { ...props } />
```

Comment:
I would destructure className in the argument list so you're not passing through `props.className`.  

```js
function TextToolbarButton( { className, ...otherProps } ) {
    const classes = classnames(
		'wc-block-text-toolbar-button',
		className
	);
	return (
		<Button className={ classes } { ...otherProps } />
}
```